specification/dimensions/dimensions.mdpp

```diff
@@ -13,6 +13,7 @@ dimensions to categorize, filter, and reveal details in your data when grouped w
 !INCLUDE "chargeperiodend.md",1
 !INCLUDE "chargeperiodstart.md",1
 !INCLUDE "chargetype.md",1
+!INCLUDE "commitmentid.md",1
```

Comment:
change file name to commitmentdiscountid if @flanakin 's suggestion goes ahead.

Reply:
Updated with the latest commit.